README.md

```diff
@@ -0,0 +1,121 @@
+# Majority Judgment for Typescript
+
+[![MIT](https://img.shields.io/github/license/MieuxVoter/majority-judgment-library-typescript?style=for-the-badge)](LICENSE)
+[![Release](https://img.shields.io/github/v/release/MieuxVoter/majority-judgment-library-typescript?include_prereleases&style=for-the-badge)](https://github.com/MieuxVoter/majority-judgment-library-typescript/releases)
+[![Build Status](https://img.shields.io/github/workflow/status/MieuxVoter/majority-judgment-library-typescript/typescript?style=for-the-badge)](https://github.com/MieuxVoter/majority-judgment-library-typescript/actions/workflows/typescript.yml)
+[![Code Quality](https://img.shields.io/codefactor/grade/github/MieuxVoter/majority-judgment-library-typescript?style=for-the-badge)](https://www.codefactor.io/repository/github/mieuxvoter/majority-judgment-library-typescript)
+![LoC](https://img.shields.io/tokei/lines/github/MieuxVoter/majority-judgment-library-typescript?style=for-the-badge)
+[![Discord Chat](https://img.shields.io/discord/705322981102190593.svg?style=for-the-badge)](https://discord.gg/rAAQG9S)
+
+This typescript package helps to resolve polls using [Majority Judgment](https://fr.wikipedia.org/wiki/Jugement_majoritaire).
+
+## Features
+
+-   Efficient Majority Judgment algorithm, scales well to billions of participants
+-   Configure whether to favor _adhesion_ or _contestation_ (default)
+-   Balance proposal tallies using a static default grade or the median grade
+-   Room for Central Judgment and Usual Judgment
+-   Unit-tested (run `npm run converage`)
+
+## Install
+
+`npm install majority-judgment`
+
+## Get started
+
+### Random sample
```

Comment:
The purpose of this example is not about sampling a ballot, but about displaying TallyCollector's usage. So I would rather call it "Majority judgment from ballots"